### ISSUE TITLE
[Bug Report] reading rich editbox in NVDA and Chrome

### ISSUE THREAD

**Issue by khsbory** [nvaccess/nvda#7755]
Hello, I'm Hyongsop Kim in Korea.
I want to report one issue regarding rich multi line edit box within chrome browser with NVDA most next version.
In chrome, once I type several lines in general text area multi line edit box with some blank line, if I read through lines with arrow keys, there is no problem in NVDA next version.
But in rich edit box, like gmail, once I type several lines with some blank lines, if I read lines with my arrow keys, NVDA doesn't read the line that I located correctly.
In other words, if I press down arrow key and it is blank line, NVDA should say blank, but it says previous line.
You can test it with Gmail's compose email page.
in rich edit, you can type 4 lines.
First line, you can type one sentence.
Second line, blank itself.
Third line, you can type another one sentence.
Last line, blank itself.
And then, you can read these lines with your arrow keys.
You may find strange reading behavior such as reading previous or next line between blank lines.
Please fix this issue.

If you need additional information, please let me know.

thank you.

**Comment by bhavyashah** [comment]
Confirmed using other rich edit fields such as the one on Quora and
several versions of Chrome and NVDA. Given the commonness of this
issue, I am positive that a duplicate might already be in existence on
the tracker, possibly concerning another manifestation of this same
buggy behaviour.

On 11/14/17, khsbory <notifications@github.com> wrote:
> Hello, I'm Hyongsop Kim in Korea.
> I want to report one issue regarding rich multi line edit box within chrome
> browser with NVDA most next version.
> In chrome, once I type several lines in general text area multi line edit
> box with some blank line, if I read through lines with arrow keys, there is
> no problem in NVDA next version.
> But in rich edit box, like gmail, once I type several lines with some blank
> lines, if I read lines with my arrow keys, NVDA doesn't read the line that I
> located correctly.
> In other words, if I press down arrow key and it is blank line, NVDA should
> say blank, but it says previous line.
> You can test it with Gmail's compose email page.
> in rich edit, you can type 4 lines.
> First line, you can type one sentence.
> Second line, blank itself.
> Third line, you can type another one sentence.
> Last line, blank itself.
> And then, you can read these lines with your arrow keys.
> You may find strange reading behavior such as reading previous or next line
> between blank lines.
> Please fix this issue.
>
> If you need additional information, please let me know.
>
> thank you.
>
> --
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/nvaccess/nvda/issues/7755


-- 
Best Regards
Bhavya Shah

Blogger at Hiking Across Horizons: https://bhavyashah125.wordpress.com/

Contacting Me
E-mail Address: bhavya.shah125@gmail.com
Follow me on Twitter @BhavyaShah125 or www.twitter.com/BhavyaShah125
Mobile Number: +91 7506221750

**Comment by ehollig** [comment]
Possibly related, #5115. Although that deals with Firefox, and this is specifically
 discussing Chrome. Sounds as if the issues are similar though.

**Comment by Adriani90** [comment]
@bhavyashah, @khsbory I am testing with Chrome 73 and NVDA 2019.1. I cannot reproduce this issue. Are you still able to reproduce it?

**Comment by Adriani90** [comment]
We do not have any updates from the author since more than one year. I close this as works for me. Please feel free to comment on it if you are still having this problem and we can reopen it.
